### PR TITLE
Project Page search bar through project name also added

### PR DESCRIPTION
**Pull request by AmeyaJain-25**
Issue #440 Resolved
Initially the search bar in project page was only searching for tags.
Now, the changes made to search through project name also.

Searching through tag name:
image

Searching through project name TOO:
image

### CODE REVIEW

**Comment by AmeyaJain-25** [comment]
The check for greetings is not done. I'm new to this and don't know how to resolve this.
Please help

**Comment by smaranjitghose** [comment]
> The check for greetings is not done. I'm new to this and don't know how to resolve this.
> Please help

It's a bug in the github actions. You can ignore it

File: projects.html [hunk 1]
```diff
@@ -1,95 +1,160 @@
...
-  <meta name="twitter:card" content="summary">
-  <meta name="twitter:site" content="https://portfolio.smaranjitghose.codes/projects.html">
+    <meta name="application-name" content="John Doe | Projects" />
+    <meta name="apple-mobile-web-app-title" content="John Doe | Projects" />
+    <meta
+      name="summary"
+      content="An open source project aimed at providing free and beautiful templates to everyone for building their portfolio websites and showcase their work to the world."
+    />
+    <link
+      rel="stylesheet"
```
Comment (smaranjitghose) [inline projects.html:53]: Avoid these indentation and formatting changes till the end else others will face merge conflicts😊

**Review by smaranjitghose — APPROVED** [review]

File: projects.html (hunk 1 above)
Comment (AmeyaJain-25) [reply projects.html:53]: Ok sure.
I'll take care of this

**Pull request merged**